### ISSUE TITLE
[FEA] Add post_block layer to MultiClassClassificationTask class

### ISSUE THREAD

**Issue by rnyak** [NVIDIA-Merlin/models#468]
# 🚀 Feature request

Currently we get logits as outputs if we train a `MultiClassClassificationTask()` model.  It'd be useful to add an optinal post-block layer in the MultiClassClassificationTask class and set it to softmax layer as default. That way users can generate predict probas as well.

**Comment by rnyak** [comment]
in the prediction mode we can make the softmax as default to get the probas.
